gsdk/tests/rpc.rs

```diff
@@ -15,24 +15,72 @@
 //
 // You should have received a copy of the GNU General Public License
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
+#![cfg(feature = "testing")]
+
+use gear_core::ids::{CodeId, ProgramId};
+use gsdk::{
+    ext::{sp_core::crypto::Ss58Codec, sp_runtime::AccountId32},
+    testing::Node,
+    Api, Result,
+};
+use parity_scale_codec::Encode;
+
+/// AccountId32 of Alice
+fn alice_account_id() -> AccountId32 {
+    AccountId32::from_ss58check("5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY")
+        .expect("Invalid address")
+}
+
+/// Generate program id from code id and salt
+fn program_id(bin: &[u8], salt: &[u8]) -> ProgramId {
+    ProgramId::generate(CodeId::generate(bin), salt)
+}
+
+/// Websocket address of the node.
+fn ws(node: &Node) -> String {
+    format!("ws://{}", node.address())
+}
```

Comment:
Do you want to move this into `tests/common/mod.rs` and use it in the `NodeExec` impl as well as somewhere else

Reply:
Maybe `program_id` also can be moved into the common place for possible re-use throughout the tests